lib/core/interactor/corporate_reader.rb

```diff
@@ -0,0 +1,9 @@
+module Core
+  class CorporateReader < ArticleReader
+    private
```

Comment:
@dougdroper Like we discussed, I inherit from Article Reader. There are some interactors, controllers and views with same concept duplicated. But this refactor will be done in a diff and focused PR.